static/app/stories/view/storyFooter.tsx

```diff
@@ -14,11 +16,24 @@ export function StoryFooter() {
   const {story} = useStory();
   const stories = useStoryBookFilesByCategory();
   const pagination = findPreviousAndNextStory(story, stories);
+  const organization = useOrganization();
+
+  const {state: prevState, ...prevTo} = pagination?.prev?.location ?? {};
+  const {state: nextState, ...nextTo} = pagination?.next?.location ?? {};
 
   return (
     <Flex align="center" justify="between" gap="xl">
       {pagination?.prev && (
-        <Card to={pagination.prev.location} icon={<IconArrow direction="left" />}>
+        <Card
+          to={{
+            ...prevTo,
+            pathname: normalizeUrl(
+              `/organizations/${organization.slug}${pagination.prev.location.pathname}`
+            ),
+          }}
+          state={prevState}
```

Comment:
isn’t `state` just a “hidden” value we can pass between links? Could we also use a query param for this, as I guess `state` wouldn’t work when opening the link in a new tab.

I actually like that we can’t have `state` on links now, I feel it’s too constrained to work with, and we can mostly just use url params

Reply:
@TkDodo yes it is. I think now that this is all using the correct links, and the links work in both devserver and dev-ui, I can go and update this to use query params instead of state. I will open a stacked PR on top of this one, and we can get that merged in before we merge this.